CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # PowerShell Editor Services Release History
 
+## v2.0.0-preview.1
+### Wednesday, January 23, 2019
+
+#### Preview builds of PowerShell Editor Services are now available
+
+#### What the first preview contains
+
+The v2.0.0-preview.1 version of the extension is built on .NET Standard (enabling support for both Windows PowerShell and PowerShell Core from one assembly)
+
+It also contains PSReadLine support in the integrated console for Windows behind a feature flag. PSReadLine provides a consistent and rich interactive experience, including syntax coloring and multi-line editing and history, in the PowerShell console, in Cloud Shell, and now in VSCode terminal. For more information on the benefits of PSReadLine, check out their [documentation](https://docs.microsoft.com/en-us/powershell/module/psreadline/about/about_psreadline?view=powershell-6).
+
+To enable PSReadLine support in the Preview version on Windows, please add the following to your user settings:
```

Comment:
This bit is quite VSCode specific. May be worth saying `-FeatureFlags @('PSReadLine')` instead